askama_derive/src/generator/expr.rs

```diff
@@ -458,7 +459,7 @@ impl<'a> Generator<'a, '_> {
         for _ in 0..*refs {
             buf.write('&');
         }
-        self.visit_path(buf, path);
+        self.visit_macro_path(buf, path);
```

Comment:
I'm wondering if I should allow `TyGenerics` to have path with generics in them... No limit to generics recursion. =D